src/accessibility/RovingTabIndex.tsx

```diff
@@ -131,6 +131,7 @@ export const reducer = (state: IState, action: IAction) => {
         }
 
         case Type.SetFocus: {
+            if (state.activeRef === action.payload.ref) return state;
```

Comment:
We should add a comment on why this is necessary or an example of what situation it fixes

Reply:
It doesn't fix anything, just a short circuit condition to prevent re rendering when nothing changes, will comment though.